ProjectBank/Server.Tests/Controllers/MyProjectsControllerTests.cs

```diff
@@ -0,0 +1,20 @@
+namespace Server.Tests;
+
+public class MyProjectsControllerTests
+{
+    [Fact]
+    public async Task Get_returns_MyProjects_from_repo()
+    {
+        // Arrange
+        var expected = Array.Empty<ProjectDTO>();
+        var repository = new Mock<IProjectRepository>();
+        repository.Setup(m => m.ReadProjectsBySupervisorIdAsync("1")).ReturnsAsync(expected);
+        var controller = new MyProjectsController(repository.Object) { GetObjectId = () => "1" };
+
+        // Act
+        var actual = await controller.Get();
+
+        // Assert
+        Assert.Equal(expected, actual);
```

Comment:
So we expect no projects in MyProject? isnt that a bit boring and easy compared to actually adding projects to the db and then requesting projects by this supervisor?

Reply:
I don't think this can be done since we are mocking the IProjectsRepository, meaning that we never actually add things to the DB. We also never establish the connection to the DB from the tests meaning that we can't add some projects to the supervisor with ID 1 since it would never go in to the DB. Also doing this would be out of scope since we test these things in the infrastructure tests :)